cmd/broker/fanout/main.go

```diff
@@ -51,8 +51,11 @@ type envConfig struct {
 	MaxStaleDuration time.Duration `envconfig:"MAX_STALE_DURATION" default:"1m"`
 
 	// MaxOutstandingBytes is the maximum size of unprocessed messages (unacknowledged but not yet expired).
-	// Default is 800Mb (~763Mi)
-	MaxOutstandingBytes int `envconfig:"MAX_OUTSTANDING_BYTES" default:"800000000"`
+	// Default is 400Mb
+	MaxOutstandingBytes int `envconfig:"MAX_OUTSTANDING_BYTES" default:"400000000"`
```

Comment:
For future reference, #1762 is now updated to reflect some additional evaluation of this.